GoogleDataTransport/GDTCORLibrary/GDTCORConsoleLogger.m

```diff
@@ -25,12 +25,12 @@
 
 void GDTCORLog(GDTCORMessageCode code, NSString *format, ...) {
 // Don't log anything in not debug builds.
-#ifndef NDEBUG
+#if !NDEBUG
```

Comment:
The DEBUG macro is not defined by any standard, whereas NDEBUG is. Generally, NDEBUG should be preferred as it can be more readily relief upon.

https://stackoverflow.com/a/17241278